src/Controls/src/Core/Page/Page.cs

```diff
@@ -261,6 +263,12 @@ Thickness ISafeAreaView2.SafeAreaInsets
 			}
 		}
 
+		/// <inheritdoc cref="ISafeAreaView2.IgnoreSafeAreaForEdge"/>
+		bool ISafeAreaView2.IgnoreSafeAreaForEdge(int edge)
```

Comment:
For page the default for IgnoreSafeAreaForEdge should be "All" if the user hasn't specified anything